src/EnergyPlus/Humidifiers.cc

```diff
@@ -1189,8 +1185,6 @@ namespace Humidifiers {
         Real64 SteamSatEnthalpy;        // enthalpy of saturated steam at 100C [J/kg]
         Real64 WaterSatEnthalpy;        // enthalpy of saturated water at 100C [J/kg]
         Real64 Tref;                    // humidifier entering water temperature [C]
-        int RefrigerantIndex;           // refiferant index
-        int WaterIndex;                 // fluid type index
```

Comment:
[src/EnergyPlus/Humidifiers.cc:1192]:(style),[variableScope],The scope of the variable 'RefrigerantIndex' can be reduced.
[src/EnergyPlus/Humidifiers.cc:1193]:(style),[variableScope],The scope of the variable 'WaterIndex' can be reduced.